### PR TITLE
Allow deploying nova-controller on SLE12 nodes

### PR DESCRIPTION
**Pull request by vuntz**

### CODE REVIEW

File: chef/cookbooks/nova/recipes/memcached.rb [hunk 1]
```diff
@@ -19,10 +19,11 @@
 
 package "memcached"
 
-if node.platform_family? "rhel"
-  package "python-memcached"
-elsif node.platform_family? "suse"
-  package "python-python-memcached"
+case node[:platform_family]
+  when 'rhel'
```
Comment (tboerger) [inline chef/cookbooks/nova/recipes/memcached.rb]: Mixed usage of single and double quotes?

File: chef/cookbooks/nova/recipes/memcached.rb [hunk 2]
```diff
@@ -19,10 +19,11 @@
...
+    package "python-memcached"
+  when 'suse'
```
Comment (tboerger) [inline chef/cookbooks/nova/recipes/memcached.rb]: Mixed usage of single and double quotes?

**Comment by tboerger** [comment]
Small nitpick, otherwise +1

**Comment by vuntz** [comment]
Tempest is passing.

**Comment by rsalevsky** [comment]
+1

File: chef/cookbooks/nova/recipes/memcached.rb (hunk 1 above)
Comment (vuntz) [inline chef/cookbooks/nova/recipes/memcached.rb]: fixed

**Comment by tboerger** [comment]
+1

**Pull request merged**